cumulusci/cli/cci.py

```diff
@@ -809,6 +807,19 @@ def org_connect(config, org_name, sandbox, login_url, default, global_org):
     oauth_dict = oauth_capture()
     org_config = OrgConfig(oauth_dict, org_name)
     org_config.load_userinfo()
+    org_config._load_orginfo()
+    if (
+        org_config.organization_sobject
+        and "TrialExpirationDate" in org_config.organization_sobject.keys()
+    ):
+        if org_config.organization_sobject["TrialExpirationDate"] is None:
+            org_config.config["expires"] = None
+        else:
+            org_config.config["expires"] = parse_api_datetime(
+                org_config.organization_sobject["TrialExpirationDate"]
+            )
```

Comment:
You can see in the coveralls report that lines 815-820 are not covered by the existing tests. It looks like we need to add a test for org_connect in the scenario where the org has a TrialExpirationDate.

Reply:
Was going to ask you about this!